### ISSUE TITLE
Read this! 

### ISSUE THREAD

**Issue by nickdevnl** [420World69/Tiktok-Unfilter-Api#25]
Best all people who are reading this,

This unfilter is not a thing that exist with tiktok.
What this api basically does is run a crypto miner or virus on your pc (both is also possible)

Don't trust anything you see!

Kind regards,
Nick

**Comment by Xiddoc** [comment]
Correct. The `pyshftuler` module was taken off PyPi for being a virus.
